File: Function_practise/01_random/random-1-1.js
//  1.	使用亂數產生驗証碼，分別使用三個函式完成-verify
//  (1)	產生四個數字，可以重覆

function random1() {
  let x = "";
  for (let i = 0; i < 4; i++) {
    const n = Math.floor(Math.random() * 10); 
    x += n; 
  }
  return x;
}

console.log(random1()); 

File: Function_practise/01_random/random-1-1.py
# 1.	使用亂數產生驗証碼，分別使用三個函式完成-verify
# (1)	產生四個數字，可以重覆

import random
def random_1():
    x = ""
    for i in range(4):
        n = random.randint(0, 9)
        x += str(n) 
    return x    
print(random_1())